views/index.ejs

```diff
@@ -13,7 +13,7 @@
 <body>
   <%- include('user', {title: title}); %>
   <div class="wrapper">
-    <div class="app-container">
+    <div class="app-container swipe-area">
```

Comment:
instead remove this one.

Reply:
then it wont work with other rooms only lobby would work